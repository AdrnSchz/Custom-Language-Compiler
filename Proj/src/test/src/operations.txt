CEO ยก
    bro res = 6 + 15 / 3 * 8 + 7.
!