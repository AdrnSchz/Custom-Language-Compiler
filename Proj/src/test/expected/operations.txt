	.data
TRUE: .float   1
	.text
	.data
FALSE: .float   0
	.text
	li   $fp, 2147483644
	move $sp, $fp
	j    _CEO
_CEO:
	sub  $sp, $sp, 4
	li   $t0, 6
	sub  $sp, $sp, 4
	li   $t1, 15
	sub  $sp, $sp, 4
	li   $t2, 3
	sub  $sp, $sp, 4
	div  $t1, $t2
	mflo $t4
	move $t3, $t4
	li   $t4, 8
	sub  $sp, $sp, 4
	mul  $t6, $t3, $t4
	move $t5, $t6
	li   $t6, 7
	sub  $sp, $sp, 4
	add  $t8, $t5, $t6
	move $t7, $t8
	add  $t9, $t0, $t7
	move $t8, $t9
	lw   $t9, -4($fp)
	move $t9, $t8
	li   $v0, 10
	syscall